template/map.jinja

```diff
@@ -5,18 +5,18 @@
 {% import_yaml 'template/defaults.yaml' as defaults %}
 {% import_yaml 'template/osfamilymap.yaml' as osfamilymap %}
 {% import_yaml 'template/osmap.yaml' as osmap %}
+{% import_yaml 'template/osfingermap.yaml' as osfingermap %}
 
-{## Merge the osfamilymap ##}
-{% set osfamily = salt['grains.filter_by'](osfamilymap, grain='os_family') or {} %}
-{% do salt['defaults.merge'](defaults['template'], osfamily) %}
-
-{## Merge the osmap ##}
-{% set os = salt['grains.filter_by'](osmap, grain='os') or {} %}
-{% do salt['defaults.merge'](defaults['template'], os) %}
-
-{## Merge the lookup ##}
-{% set lookup = salt['pillar.get']('template:lookup', default={}) %}
-{% do salt['defaults.merge'](defaults['template'], lookup) %}
+{% set defaults = salt['grains.filter_by'](default_settings,
```

Comment:
Because of this change here.

Reply:
> @alxwr Just need this last fix and we're good to go, thanks!

Should be fixed now. Thanks for reviewing!